### ISSUE TITLE
Show the username of the current user

### ISSUE THREAD

**Issue by JonathanBout** [filebrowser/filebrowser#2692]
**Is your feature request related to a problem? Please describe.**
When I'm logged in in a specific account I can't see which one. As I have an admin account and a personal account it's difficult to see which I'm logged in to.

**Describe the solution you'd like**
Show the username of the current user somewhere in the header or in the sidebar.

**Describe alternatives you've considered**
None

**Comment by JonathanBout** [comment]
I'm working on this myself

**Comment by JonathanBout** [comment]
Does anyone have suggestions where to add the username? We have a few options:
- In the header, but that might break the layout on mobile
- We can add a new label in the sidebar with the username and maybe an icon
- We can change the logout button to say for example 'Logged in as \<username\>'.  

I think I'm going for the label in the sidebar, but if you can come up with a better location, lmk :)

**Comment by JonathanBout** [comment]
_I'm working on it, give me some time_

**Comment by JonathanBout** [comment]
_ping_

**Comment by JonathanBout** [comment]
And another ping

**Comment by JonathanBout** [comment]
Unstale... Again I guess

**Comment by niubility000** [comment]
Have you finished? I made it 2 years ago.![image](https://github.com/filebrowser/filebrowser/assets/76441520/e7d2757e-f882-4ded-9e3d-60ad3aa9f9bb)
The user name is aaa.

**Comment by JonathanBout** [comment]
@niubility000 I made a pull request with my changes: https://github.com/filebrowser/filebrowser/pull/2821

**Comment by niubility000** [comment]
> @niubility000 I made a pull request with my changes: https://github.com/filebrowser/filebrowser/pull/2821

It's better than mine.

**Comment by JonathanBout** [comment]
> It's better than mine.

Thank you!

**Comment by JonathanBout** [comment]
Unstale 😐

**Comment by JonathanBout** [comment]
Unstale...

**Comment by JonathanBout** [comment]
unstale

**Comment by JonathanBout** [comment]
u n s t a l e

**Comment by JonathanBout** [comment]
_u n s t a l e_

**Comment by aae42** [comment]
no

**Comment by JonathanBout** [comment]
🐌

**Comment by JonathanBout** [comment]
.

**Comment by JonathanBout** [comment]
.

**Comment by JonathanBout** [comment]
.

**Comment by xiaobaidadada** [comment]
https://github.com/xiaobaidadada/filecat support custom_fun_opt

**Comment by JonathanBout** [comment]
.

**Comment by xiaobaidadada** [comment]
https://github.com/xiaobaidadada/filecat can do base filebrowser

**Comment by JonathanBout** [comment]
.

**Comment by JonathanBout** [comment]
.

**Comment by christaikobo** [comment]
You are one persistent man....
Try this fork it does what you want. https://github.com/gtsteffaniak/filebrowser

![Image](https://github.com/user-attachments/assets/d12729df-d481-41d3-8b7a-1df247501f9f)

**Comment by JonathanBout** [comment]
@christaikobo thank you for the suggestion! I don't think it is possible to install this to OpenMediaVault without manually configuring Docker, right? With filebrowser/filebrowser I can do that.

But well, this is taking so long I could've completed that setup a year ago already 😅

**Comment by christaikobo** [comment]
Yeah you need to use docker. It's not too complicated to be honest.

**Comment by jniggemann** [comment]
Thank you for opening an issue and contributing to the filebrowser project!
While we're looking for maintainers, we limited the issue tracker to actual bugs. Feature requests are now tracked in the  [discussions](https://github.com/filebrowser/filebrowser/discussions).
Your idea is valuable for us and as such, we're moving it there.


> [!WARNING]  
> This project is currently on maintenance-only mode, and is looking for new maintainers. For more information, please read the discussion #4906. Therefore, please note the following:
>
>- It can take a while until someone gets back to you. Please be patient.
>- Issues are only being used to track bugs. Any unrelated issues will be converted into a discussion.
>- No new features will be implemented until further notice. The priority is on triaging issues and merge bug fixes.
>
>If you're interested in maintaining this project, please reach out via the discussion above.
